src/ProgramSearchLogViewsData.php

```diff
@@ -4,7 +4,11 @@
 
 use Drupal\views\EntityViewsData;
 
+/**
+ *
```

Comment:
Description required. Like a "Class to provide visitors counter for Program Search Log Views""